src/main/java/org/fundacionjala/pivotal/pages/Header.java

```diff
@@ -115,7 +115,6 @@ public String getTitleWSName() {
      **/
     public void openMenu() {
         this.action.click(this.projecMenu);
-        final int timeToWait = 3000;
-        this.action.pause(timeToWait);
+        this.action.pause(3000);
```

Comment:
As happens belong other pause action, consider to use a global variable, a `WS_TIME`